cli/js/fetch.ts

```diff
@@ -332,6 +436,24 @@ export class Response implements domTypes.Response {
       this.body
     );
   }
+
+  redirect(url: URL | string, status: number): domTypes.Response {
+    if (![301, 302, 303, 307, 308].includes(status)) {
+      throw new RangeError(
+        "The redirection status must be one of 302, 302, 303, 307 and 308."
```

Comment:
I know this is super picky, and sorry for that, but shouldn't the range of the error code be `301, 302, 303, 307 and 308`? Or have I completely missed something?

Reply:
Oh yes, I must have missed that one. I think that at some point git might have lost some of my changes that weren't committed, so I'm just working on getting everything to build again.

Reply:
Sure thing man. This isn't urgent or anything. Just be shame if it slipped through and somebody else had to go back and fix it.